bot/utils/redis_cache.py

```diff
@@ -21,7 +21,37 @@ class communicates with the Redis server.
 
     We implement several convenient methods that are fairly similar to have a dict
     behaves, and should be familiar to Python users. The biggest difference is that
-    all the public methods in this class are coroutines.
+    all the public methods in this class are coroutines, and must be awaited.
+
+    Because of limitations in Redis, this cache will only accept strings, integers and
+    floats both for keys and values.
+
+    Simple example for how to use this:
+
+    class SomeCog(Cog):
+        # To initialize a valid RedisCache, just add it as a class attribute here.
+        # Do not add it to the __init__ method or anywhere else, it MUST be a class
+        # attribute. Do not pass any parameters.
+        cache = RedisCache()
+
+        async def my_method(self):
+            # Now we can store some stuff in the cache just by doing this.
+            # This data will persist through restarts!
+            await self.cache.set("key", "value")
```

Comment:
Maybe we could add that it's important to access it like this, on the instance of the cog, with `self.cache`. That's vital the very first time, as we need to grab the `Bot` reference of `self` (the cog instance).

Reply:
Done!